config/Clojure-replUiLabels.xml

```diff
@@ -40,4 +40,24 @@ under the License.
         <value xml:lang="zh">不允许你浏览这个页面。</value>
         <value xml:lang="zh-TW">不允許您檢視這個頁面.</value>
     </property>
+    <property key="clojureReplstartRepl">
```

Comment:
Folosește sintaxa cu @component .... @ ca in celelalte fișiere UI din OFBiz .

Reply:
am adaugat sintaxa @component-resource-name@ pentru UiLabels